shell-friends.md

```diff
@@ -176,7 +176,7 @@ Explanation
 
 Add this function to your ~/.bashrc are reload your shell (or open a new shell).
 You'll need to substitute these database credentials with your own.
-When you need create a new wordpress install, call this function and specify the
+When You'll need to create a new site and install wordpress, call this function and specify the
```

Comment:
Your suggestion has other issues as well, but the original sentence is indeed missing a `to` and has wrong casing for WordPress.

Here's what I think is a good fix:
```suggestion
When you need to create a new WordPress site, call this function and specify the
```